pkg/scheduler/framework/plugins/interpodaffinity/scoring_test.go

```diff
@@ -581,7 +582,9 @@ func TestPreferredAffinity(t *testing.T) {
 		t.Run(test.name, func(t *testing.T) {
 			state := framework.NewCycleState()
 			snapshot := cache.NewSnapshot(test.pods, test.nodes)
+			f, _ := frameworkruntime.NewFramework(nil, nil, nil)
 			p := &InterPodAffinity{
+				parallelizer: f.Parallelizer(),
```

Comment:
well.... you can could create the parallelizer directly.

But even better, can you call `New` instead?

Reply:
Done.